### PR TITLE
Create animepahe.json

### PR DESCRIPTION
**Pull request by MulverineX**
image

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
A lot of your strings are unneeded as they are in the general.json file

File: src/Presence/animepahe.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "description": ""
+  },
+  "animepahe.browsingHome": {
+    "message": "Latest Releases",
+    "description": ""
+  },
+  "animepahe.browsingAll": {
+    "message": "A-Z",
+    "description": ""
+  },
```
Comment (Bas950) [inline src/Presence/animepahe.json:13]: Seems unneeded to me

File: src/Presence/animepahe.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "animepahe.browsingSeason": {
+    "message": "Season",
+    "description": "TV show/series season eg. Season 5 of The Flash"
+  },
+  "animepahe.browsingSpecial": {
+    "message": "Special",
+    "description": ""
+  },
```
Comment (Bas950) [inline src/Presence/animepahe.json:21]: add description

File: src/Presence/animepahe.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  "animepahe.view": {
+    "message": "View on",
+    "description": "followed by Pahe/AniList"
+  },
```
Comment (Bas950) [inline src/Presence/animepahe.json:25]:
```suggestion
  "animepahe.view": {
    "message": "View on {0}",
    "description": "{0} will be replaced by Pahe/AniList"
  },
```

File: src/Presence/animepahe.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  "animepahe.watch": {
+    "message": "Watch",
+    "description": "followed by Movie or episode"
+  },
```
Comment (Bas950) [inline src/Presence/animepahe.json:29]: image
Already strings for that available

File: src/Presence/animepahe.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  "animepahe.movie": {
+    "message": "Movie",
+    "description": ""
+  },
```
Comment (Bas950) [inline src/Presence/animepahe.json:33]: Content?

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: src/Presence/animepahe.json (hunk 4 above)
Comment (MulverineX) [reply src/Presence/animepahe.json:29]: There's no "Watch Movie"

File: src/Presence/animepahe.json (hunk 1 above)
Comment (MulverineX) [reply src/Presence/animepahe.json:13]: What about languages that are not latin based

File: src/Presence/animepahe.json (hunk 5 above)
Comment (MulverineX) [reply src/Presence/animepahe.json:33]: What?

File: src/Presence/animepahe.json (hunk 1 above)
Comment (Bas950) [reply src/Presence/animepahe.json:9]: Still seems unneeded to me

File: src/Presence/animepahe.json (hunk 4 above)
Comment (Bas950) [reply src/Presence/animepahe.json:25]: Then add it to general.json as it can be used more often

File: src/Presence/animepahe.json (hunk 5 above)
Comment (Bas950) [reply src/Presence/animepahe.json:29]: Context*

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: src/Presence/animepahe.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
+{
+  "animepahe.browsingHome": {
```
Comment (Bas950) [inline src/Presence/animepahe.json:2]:
```suggestion
  "animepahe.latestRelease": {
```

File: src/Presence/animepahe.json [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+    "message": "Latest Releases",
+    "description": "Recently released content, can be new or old anime, its just new to the site"
+  },
+  "animepahe.browsingSeason": {
+    "message": "Season",
+    "description": "TV show/series season eg. Season 5 of The Flash"
+  },
+  "animepahe.browsingSpecial": {
```
Comment (Bas950) [inline src/Presence/animepahe.json:10]:
```suggestion
  "animepahe.special": {
```

File: src/Presence/animepahe.json [hunk 8]
```diff
@@ -0,0 +1,22 @@
+{
+  "animepahe.browsingHome": {
+    "message": "Latest Releases",
+    "description": "Recently released content, can be new or old anime, its just new to the site"
+  },
+  "animepahe.browsingSeason": {
```
Comment (Bas950) [inline src/Presence/animepahe.json:6]:
```suggestion
  "animepahe.season": {
```

File: src/Presence/animepahe.json [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+    "message": "Season",
+    "description": "TV show/series season eg. Season 5 of The Flash"
+  },
+  "animepahe.browsingSpecial": {
+    "message": "Special",
+    "description": "A type of related content; a special release episode not included in the original release often with differing style"
+  },
+  "animepahe.view": {
+    "message": "View on {0}",
+    "description": "{0} will be replaced by Pahe or AniList"
```
Comment (Bas950) [inline src/Presence/animepahe.json:16]:
```suggestion
    "description": "{0} will be replaced by \"Pahe\" or \"AniList\""
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
That should be it

**Pull request merged**